astroquery/nasa_exoplanet_archive/tests/test_nasa_exoplanet_archive.py

```diff
@@ -7,26 +7,32 @@
 from urllib.parse import urlencode
 
 import astropy.units as u
+from astropy.io.ascii import write as ap_write
 import numpy as np
 import pkg_resources
 import pytest
 import requests
+import pyvo
 from astropy.coordinates import SkyCoord
 from astropy.tests.helper import assert_quantity_allclose
 from astropy.utils.exceptions import AstropyDeprecationWarning
 
 from ...exceptions import InputWarning, InvalidQueryError, NoResultsWarning
 from ...utils.testing_tools import MockResponse
-from ..core import NasaExoplanetArchive, conf
+from ..core import NasaExoplanetArchive, conf, InvalidTableError
 
 MAIN_DATA = pkg_resources.resource_filename("astroquery.nasa_exoplanet_archive", "data")
 TEST_DATA = pkg_resources.resource_filename(__name__, "data")
 RESPONSE_FILE = os.path.join(TEST_DATA, "responses.json")
+os.environ["NASA_EXOPLANET_ARCHIVE_GENERATE_RESPONSES"] = '1'  # Activate for generating responses
```

Comment:
This shouldn't be set here. This environment variable should *only* be set on your local machine when *creating* the tests. This was a hack that I used to automatically generate the correct mocked responses, but shouldn't ever be used when running the tests.